probe/endpoint/reporter.go

```diff
@@ -157,6 +158,10 @@ func (r *Reporter) Report() (report.Report, error) {
 				fromNodeInfo[report.HostNodeID] = hostNodeID
 			}
 
+			if conn.Proc.NetNamespaceID > 0 {
```

Comment:
Actually, it seems inodes cannot be zero: http://stackoverflow.com/questions/2099121/why-do-inode-numbers-start-from-1-and-not-0